labelCloud/definitions/label_format.py

```diff
@@ -0,0 +1,14 @@
+from enum import Enum
+from typing import List
+
+
+class LabelFormat(str, Enum):
+    VERTICES = "vertices"
+    CENTROID_REL = "centroid_rel"
+    CENTROID_ABS = "centroid_abs"
+    KITTI = "kitti"
+    KITTI_UNTRANSFORMED = "kitti_untransformed"
+
+    @classmethod
+    def list(cls) -> List["LabelFormat"]:
```

Comment:
It might be possible to use the `__contain__` built-in [method](https://docs.python.org/3/library/enum.html#enum.EnumType.__contains__)

Reply:
Good to know but I also need a list of the enum values here:
https://github.com/ch-sa/labelCloud/pull/136/files#diff-f5dce3b6c97ce32bfe4da7455e45072ab1919535b050ce70c8529f50027766faR55